test/ui-test/src/constants/user_account.py

```diff
@@ -0,0 +1,2 @@
+LOGIN = ''
+PASSWORD = 'TesTEr16843/!@00'
```

Comment:
I propose that we use a simpler `1234567890` password for tests where password input doesn't matter. This will simplify the manual runs I sometimes do between failed tests to inspect/debug. I don't have to type a more complex one :)

Reply:
Good Idea @stefandunca, I'm using this variable only for validation, I created a new task to fix it https://github.com/status-im/status-desktop/issues/10679